webserver/webserver/src/test/java/io/helidon/webserver/utils/SocketHttpClient.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package io.helidon.webserver.testsupport;
+package io.helidon.webserver.utils;
 
```

Comment:
Why is this package renamed?

Reply:
The package itself not renamed. I moved `io.helidon.webserver.testsupport.SocketHttpClient` to `io.helidon.webserver.utils.SocketHttpClient` to cut a cyclic dependency.

Currently webserver/webserver-netty depends on webserver/test-support, combining webserver-netty into webserver/webserver because some of the test use LoggingTestUtils and SocketHttpClient.

I removed the usage of LoggingTestUtils because that was not really used by the tests but only by some main methods. SocketHttpClient is used only by these tests so I moved it to webserver/webserver/src/test/java under `io.helidon.webserver.utils.SocketHttpClient` in the webserver test sources.